usmqe/api/tendrlapi/cephapi.py

```diff
@@ -0,0 +1,14 @@
+"""
+Tendrl REST API for ceph.
+"""
+
+import pytest
+from usmqe.api.tendrlapi.common import TendrlApi
+
+LOGGER = pytest.get_logger("tendrlapi_ceph", module=True)
+
+
+class TendrlApiCeph(TendrlApi):
+    """ Ceph methods for Tendrl REST API.
+    """
+    pass
```

Comment:
You know that if there's a doc string, pass doesn't have to be present, don't you?